### PR TITLE
grid fix/change - * to be merged after Yazeed's PR *

### LINKED ISSUE #13: Weird looking buttons on corner pages
The height of button in corner pages should be limited.

image

### PR DESCRIPTION
**Pull request by ghost**
fixes #13 

and changes the H corner layout

The code is based on Yazeed's version of "[corner].astro" therefore to be merged after his

### CODE REVIEW

**Review by Nandini3841 — DISMISSED** [review]

**Comment by YazeedAlKhalaf** [comment]
@g9wz kindly fix the conflicts

**Comment by ghost** [comment]
fixed, @YazeedAlKhalaf please check

File: src/pages/corners/[corner].astro [hunk 1]
```diff
@@ -50,9 +52,9 @@ for (const key in rooms) {
 }
 ---
 
-<Layout title={`${corner} Classrooms`}>
```
Comment (YazeedAlKhalaf) [inline src/pages/corners/[corner].astro:53]: Revert this change :)

File: src/pages/corners/[corner].astro [hunk 2]
```diff
@@ -39,6 +40,7 @@ switch (corner) {
   case "H":
     color = ClassroomButtonColor.paleGreen;
     headerColor = HeaderColor.paleGreen;
+    gridCols = " grid-cols-1 ";
```
Comment (YazeedAlKhalaf) [inline src/pages/corners/[corner].astro:43]: Are you sure this looks good?

**Review by YazeedAlKhalaf — CHANGES_REQUESTED** [review]

File: src/pages/corners/[corner].astro (hunk 2 above)
Comment (ghost) [reply src/pages/corners/[corner].astro:43]: the original one looks dull, i had no choice

File: src/pages/corners/[corner].astro (hunk 1 above)
Comment (ghost) [reply src/pages/corners/[corner].astro:53]: i remember leaving it as it is, weird

**Review by YazeedAlKhalaf — APPROVED** [review]
LGTM

**Pull request merged**